### PR TITLE
vmui: add tip to Explore Metrics page

### PR DESCRIPTION
**Pull request by Loori-R**
- Changed tab name to `Explore Prometheus Metrics`
- Added a tip that explains the page's intended purpose
- Tip can be toggled to show/hide (state is preserved in `localStorage`)

image

#4248

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]

File: app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx [hunk 1]
```diff
@@ -38,59 +44,99 @@ const ExploreMetricsHeader: FC<ExploreMetricsHeaderProps> = ({
...
-        />
-      </div>
-    </div>
+
+      {showTips && (
+        <Alert variant={"warning"}>
+          <div className="vm-explore-metrics-header-description">
+            <p>
+              Please note: this page is solely designed for exploring Prometheus metrics.
+              Prometheus metrics always carry <code>job</code> and <code>instance</code> labels,
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx:125]:
```suggestion
              Prometheus metrics always contain <code>job</code> and <code>instance</code> labels,
```

File: app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx [hunk 2]
```diff
@@ -38,59 +44,99 @@ const ExploreMetricsHeader: FC<ExploreMetricsHeaderProps> = ({
...
+              which this page was initially created to handle.
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx:126]:
```suggestion
              and this page relies on them as filters.
```

File: app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx [hunk 3]
```diff
@@ -38,59 +44,99 @@ const ExploreMetricsHeader: FC<ExploreMetricsHeaderProps> = ({
...
+              Metrics without a <code>job</code> selection may mislead due to their diverse values.
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx:127]:
```suggestion
```

File: app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx (hunk 1 above)
Comment (hagen1778) [reply app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx:125]: Would be nice to link the convention here https://prometheus.io/docs/concepts/jobs_instances/

**Review by hagen1778 — CHANGES_REQUESTED** [review]

**Review by hagen1778 — APPROVED** [review]
Thanks!

**Pull request merged**